client/src/scripts/managers/inputManager.ts

```diff
@@ -16,6 +16,92 @@ import { FIRST_EMOTE_ANGLE, FOURTH_EMOTE_ANGLE, PIXI_SCALE, SECOND_EMOTE_ANGLE,
 import { getTranslatedString } from "../../translations";
```

Comment:
oh yeah by the way to anyone whos reading patch-notes channel i am homeless and also the fbi is after me